src/util/misc.ts

```diff
@@ -169,21 +189,39 @@ import { cos } from './index.ts';
     getBisector: function (A, B, C) {
```

Comment:
should we change the signature to reflect that A is between B and C?

Reply:
I hadn't thought of that, but it might be more intuitive